app/views/baskets/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1> Receipts History </h1>
+<div class="history-div">
+<%= render "baskets", baskets: @baskets %>
```

Comment:
Look it up, but I think there is a cool Rails convention that allows you to do `<%= render @baskets %>`